qiskit/visualization/pulse/matplotlib.py

```diff
@@ -786,6 +789,7 @@ def draw(self, schedule: ScheduleComponent,
                 All non-empty channels are shown if not provided.
             show_framechange_channels: When set `True` plot channels
                 with only framechange instructions.
+            draw_title: Add a title to the plot when set "True".
```

Comment:
```suggestion
            draw_title: Add a title to the plot when set to ``True``.
```

Reply:
Why?!

Reply:
This will display as an inline marked up `True` in the documentation.